src/Appwrite/Platform/Modules/Databases/Http/Databases/Create.php

```diff
@@ -42,7 +42,7 @@ public function __construct()
             ->label('resourceType', RESOURCE_TYPE_DATABASES)
             ->label('audits.event', 'database.create')
             ->label('audits.resource', 'database/{response.$id}')
-            ->label('sdk', new Method(
+            ->label('sdk', SDKMethod::withGridsAPI(
```

Comment:
Let's just add the two methods here instead

Reply:
Done.